app/src/main/java/com/dmonzonis/nookpendium/CollectionActivity.kt

```diff
@@ -107,34 +104,36 @@ class CollectionActivity : AppCompatActivity() {
         viewAdapter.setRecords(recordset.records)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.toolbar_menu, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
     override fun onOptionsItemSelected(item: MenuItem?): Boolean {
         // Close all drawers before to avoid having multiple drawers opening on top of each other
         closeAllDrawers()
-        return if (drawerToggle.onOptionsItemSelected(item)) {
-            true
-        } else when (item?.itemId) {
+        when (item?.itemId) {
+            android.R.id.home -> {
+                drawerLayout.openDrawer(navGamesView)
+            }
             R.id.miFilterButton -> {
-                if (drawerLayout.isDrawerOpen(navFilterMenu))
-                    drawerLayout.closeDrawer(navFilterMenu)
-                else
-                    drawerLayout.openDrawer(navFilterMenu)
-                true
+                drawerLayout.openDrawer(filterMenu)
```

Comment:
you can do both of these in a single line, no?

Reply:
sure